### ISSUE TITLE
Home screen not loading 

### ISSUE THREAD

**Issue by Arif1221** [z-huang/music#295]
java.lang.NullPointerException
	at com.zionhuang.innertube.models.response.BrowseResponse.a(SourceFile:25)
	at z6.g.b(SourceFile:4)
	at z6.g$c.t(Unknown Source:12)
	at u9.a.n(Unknown Source:8)
	at f9.h.f(SourceFile:5)
	at f9.h.d(SourceFile:2)
	at f9.h$a.n(SourceFile:5)
	at u9.a.n(Unknown Source:31)
	at f9.h.f(SourceFile:5)
	at f9.h.d(SourceFile:2)
	at f9.h$a.n(SourceFile:5)
	at u9.a.n(Unknown Source:31)
	at f9.h.f(SourceFile:5)
	at f9.h.d(SourceFile:2)
	at f9.h$a.n(SourceFile:5)
	at u9.a.n(Unknown Source:31)
	at f9.h.f(SourceFile:5)
	at f9.h.d(SourceFile:2)
	at f9.h$a.n(SourceFile:5)
	at u9.a.n(Unknown Source:31)
	at ka.p0.run(Unknown Source:86)
	at pa.j.run(Unknown Source:12)
	at qa.j.run(Unknown Source:2)
	at qa.a$a.run(SourceFile:9)

![Screenshot_2022-10-13-21-42-39-559_com.zionhuang.music.jpg](https://user-images.githubusercontent.com/45899067/195649807-a9ff656a-187c-4b69-a14b-5792d10cbef8.jpg)

**Comment by owencz1998** [comment]
Duplicate of #290 been fixed just waiting on the new version
